libr/bin/p/bin_bf.c

```diff
@@ -5,8 +5,8 @@
 #include <r_lib.h>
 #include <r_bin.h>
 
-static void *load_bytes(RBinFile *bf, const ut8 *buf, ut64 sz, ut64 loadaddr, Sdb *sdb){
-	return R_NOTNULL;
+static bool load_bytes(void **bin_obj, RBinFile *bf, const ut8 *buf, ut64 sz, ut64 loadaddr, Sdb *sdb){
```

Comment:
RBinFile should be the first argument, because it's the "this" of those methods. having this bin_obj or `bo` as 2nd argument looks good to me

Reply:
Ok. I'll swap their order then. Thanks!

Reply:
No problem. I'll swap the first 2 args order, rebase and push it again. Thanks for the review!